ui/tests/unit/serializers/node-test.js

```diff
@@ -7,11 +7,20 @@ import pushPayloadToStore from '../../utils/push-payload-to-store';
 
 moduleForSerializer('node', 'Unit | Serializer | Node', {
   needs: [
-    'serializer:node',
+    'adapter:application',
     'service:config',
+    'serializer:node',
+    'service:system',
+    'service:token',
     'transform:fragment',
     'transform:fragment-array',
+    'model:node-attributes',
+    'model:resources',
+    'model:drain-strategy',
+    'model:node-driver',
+    'model:node-event',
     'model:allocation',
+    'model:job',
```

Comment:
You'll be glad to know new testing doesn't have `needs` 🎉

Reply:
I can't wait.